woocommerce/woocommerce-android
# [Customer Search] Customer search list with pagination

Closes: #9305


### Description

The PR adds a customer list with pagination

Notice:
* The UI is not final
* Scrolling performance is not final
* Empty/Error/Loading states are coming in the next PR
* DO NOT MERGE before I change the fluxc version. The hash I use is from the PR that can not be merged yet, as test do not pass due to the fact that WC version doesn't match

### Testing instructions

* Install nightly WC build https://github.com/woocommerce/woocommerce/releases as the current doesn't have an updated search endpoint
* Using the [plugin](https://github.com/woocommerce/wc-smooth-generator) generate customers for several pages (e.g. 100+)
* Using order creation flow, open the customers screen, scroll down and notice that more customers are loaded

OR

* Ask me to invite you to the store I used
* Using order creation flow, open the customers screen, scroll down and notice that more customers are loaded

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/541d753d-08cc-4914-9ea3-2dc38790c3c8
